### PR TITLE
Update api_calls to extract_summary

### PR DESCRIPTION
**Pull request by fivetran-catfritz**
## PR Overview
**This PR will address the following Issue/Feature:**
- #137 
- #139 

**This PR will result in the following new package version:**

- v1.9.1 no schema change

**Please provide the finalized CHANGELOG entry which details the relevant changes included in this PR:**

> ## Features
> - For Fivetran Platform Connectors created after November 2024, Fivetran has deprecated the `api_call` event in favor of `extract_summary` ([release notes](https://fivetran.com/docs/logs/changelog)).
> - Accordingly, we have updated the `fivetran_platform__connector_daily_events` model to support the new `extract_summary` event while maintaining backward compatibility with the `api_call` event for connectors created before November 2024. 
> 
> ## Under the Hood
> - Replaced the deprecated `dbt.current_timestamp_backcompat()` function with `dbt.current_timestamp()` to ensure all timestamps are captured in UTC.

### Basic Validation
Please acknowledge that you have successfully performed the following commands locally:
- [x] dbt run –full-refresh && dbt test
- [x] dbt run (if incremental models are present) && dbt test

Before marking this PR as "ready for review" the following have been applied:
- [x] The appropriate issue has been linked, tagged, and properly assigned
- [x] All necessary documentation and version upgrades have been applied
- [x] docs were regenerated (unless this PR does not include any code or yml updates)
- [x] BuildKite integration tests are passing
- [x] Detailed validation steps have been provided below

### Detailed Validation
Please share any and all of your validation steps:

- Updated seed data  to make sure the values for `extract_summary` are pulled through.
  - https://github.com/fivetran/dbt_fivetran_log/blob/89a3650d3da07a29ebc8595ec194568468e22949/integration_tests/seeds/log.csv#L33-L34
  - Screenshot 2024-11-19 at 1 22 46 PM

- Consistency tests pass
  - Screenshot 2024-11-19 at 1 21 40 PM

### If you had to summarize this PR in an emoji, which would it be?
 
:dancer:

### CODE REVIEW

**Review by fivetran-jamie — APPROVED** [review]
looks great to me! just minor requests but this is approved

**Review by fivetran-catfritz — COMMENTED** [review]
Thanks @fivetran-jamie made the updates!

File: README.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
-<p align="center">
+# Fivetran Platform dbt Package ([Docs](https://fivetran.github.io/dbt_fivetran_log/))
+
+<p align="left">
```
Comment (fivetran-joemarkiewicz) [inline README.md:3]: Thanks for making this change as well!

File: models/fivetran_platform__connector_daily_events.sql [hunk 2]
```diff
@@ -7,39 +7,56 @@ with connector as (
...
 
     from {{ ref('stg_fivetran_platform__log') }}
 
-    where event_subtype in ('api_call', 
-                            'records_modified', 
-                            'create_table', 'alter_table', 'create_schema', 'change_schema_config') -- all schema changes
-                            
+    where event_subtype in (
+        'api_call', 'extract_summary', 'records_modified', 'create_table', 'alter_table',
+        'create_schema', 'change_schema_config') -- all schema changes
```
Comment (fivetran-joemarkiewicz) [inline models/fivetran_platform__connector_daily_events.sql:23]: Small note that I know isn't related to your changes directly, but is it accurate to have the comment `all schema changes` in this filter? For example, `api_call` and `extract_summary` are not inherently schema changes and we make this clear in the below `case when` statement where we categorize perform the following:
```sql
when event_subtype in ('create_table', 'alter_table', 'create_schema', 'change_schema_config') then 'schema_change'
```

If this is true, can we change this comment to be `all relevant event subtypes` or something along those lines.

**Review by fivetran-joemarkiewicz — APPROVED** [review]
LGTM with one small question/request around a comment from a previous update and if we should change it.

File: models/fivetran_platform__connector_daily_events.sql (hunk 2 above)
Comment (fivetran-catfritz) [reply models/fivetran_platform__connector_daily_events.sql]: Makes sense--updated!

**Review by fivetran-catfritz — COMMENTED** [review]
Updated.

**Pull request merged**